analytic_engine/src/compaction/mod.rs

```diff
@@ -113,6 +115,7 @@ const MIN_THRESHOLD_KEY: &str = "compaction_min_threshold";
 const MAX_THRESHOLD_KEY: &str = "compaction_max_threshold";
 const MIN_SSTABLE_SIZE_KEY: &str = "compaction_min_sstable_size";
 const TIMESTAMP_RESOLUTION_KEY: &str = "compaction_timestamp_resolution";
+const MAX_INPUT_SSTABLE_SIZE_KEY: &str = "compaction_max_input_sstable_size";
```

Comment:
I guess `compaction_max_input_sstable_size` is just a temporary method to limit the sst file size, so should we put it as one of the table options (which may be persisted as table meta data)?

Reply:
Removed